### DISCUSSION TITLE
Add FODMAP filters to all apps and content

### DISCUSSION THREAD

**Discussion by synryn** [openfoodfacts/.github#33] [Ideas]
### Problem

People with IBS do not have a good open source and free way of alleviating their condition. OpenFoodFacts could help these people with their disability and finding foods that do not flare up their disability.

**Comment by teolemon** [comment]
@synryn Can I move the discussion to the Open Food Facts server ? I have also created a quick wiki page about what I could find about FODMAP within Open Food Facts: https://wiki.openfoodfacts.org/FODMAP
It would be awesome if you could complete it.

**Comment by synryn** [comment]
The most reliable app is probably Monash University's Fodmap App, but the issue is this app is behind a paywall which limits access to people with disability. 

The general idea is that all fats and oils are fine for FODMAPs, FODMAPs are a carbohydrate issue.
